openrouteservice/src/main/java/heigit/ors/routing/graphhopper/extensions/storages/builders/HeavyVehicleGraphStorageBuilder.java

```diff
@@ -331,6 +246,19 @@ private String getHeavyVehicleValue(String key, String hv, String value)
 		
 		return null;
 	}
+
+	private void setFlagsFromTag (String tag, int flag) {
```

Comment:
May be worth just adding a small JDocs description as to what this method does so that someone can understand quickly without understanding the bit logical operators.

Reply:
Thanks Adam, fixed that!